script/upload_to_aptly

```diff
@@ -101,38 +111,25 @@ EOF
 # Make sure staging and release cache isn't mixed
 rm -Rf ~/.aptly
 
-gpg --export --armor ${GPG_KEY_ID} > ${KEY_FILE}
-aws s3 cp ${KEY_FILE} s3://${OPTION}/${KEY_NAME}
-aws s3api put-object-acl --acl public-read --bucket ${OPTION} --key ${KEY_NAME}
-
-for i in ${DIST}
-	do aptly publish drop -force-drop $i s3:${OPTION}: || true
-done
-
-for i in ${DIST}
-	do aptly repo drop -force ${i}-release || true
-done
-
-for i in ${DIST}
-	do aptly snapshot drop -force ${i}-snapshot || true
-done
-
-for i in ${DIST}
-	do aptly repo create -distribution=$i -component=main ${i}-release
-done
+gpg --export --armor "${GPG_KEY_ID}!" | aws s3 cp --acl public-read - "s3://${S3_BUCKET}/${KEY_NAME}" # key suffix ! forces gpg to use the specified key
 
-for i in ${DIST}
-	do aptly repo add ${i}-release dist/*.deb
+for i in ${DIST}; do
+	aptly publish drop -force-drop "$i" "s3:${S3_BUCKET}:" || true
+	aptly repo drop -force "${i}-release" || true
+	aptly snapshot drop -force "${i}-snapshot" || true
+	aptly repo create -distribution="$i" -component=main "${i}-release"
+	aptly repo add "${i}-release" dist/*.deb
+	aptly snapshot create "${i}-snapshot" from repo "${i}-release"
+	aptly repo edit "${i}-release"
 done
 
-for i in ${DIST}
-	do aptly snapshot create ${i}-snapshot from repo ${i}-release
-done
-
-for i in ${DIST}
-	do aptly repo edit ${i}-release
-done
+# TODO: Add some sanity checks for the repos
 
-for i in ${DIST}
-	do aptly publish snapshot -force-overwrite=true -gpg-key=${GPG_KEY_ID} ${i}-snapshot s3:${OPTION}:
+set +x
+for i in ${DIST}; do
+	if [ "${i}" = "stable" ]; then 
+        STABLE_ARGS="-origin='Brave Software' -label='Brave Browser'"
+    fi
```

Comment:
This looks like the spacing of the `if` is mismatched, I believe I see 6 spaces in front but 4 in front of the closing `fi`. (Not that this matters in bash, but my Python trained eyes noticed it just now.)